app/backend/src/couchers/servicers/blocking.py

```diff
@@ -8,6 +9,20 @@
 from proto import blocking_pb2, blocking_pb2_grpc
 
 
+def are_blocked(session, user1_id, user2_id):
```

Comment:
If you're moving this here, should we move `_relevant_user_blocks` as well?

Reply:
We might also move `are_friends` to api.py

Reply:
Import errors. Can't import models in utils, because models imports utils; this is the next best place. I just want this code to be nearby to other uses of `UserBlock` so it gets the right treatment when refactoring.

Reply:
That all makes sense. I Think we should move the other two for consistency. Doesn't really matter though.